src/usr/local/www/services_unbound.php

```diff
@@ -168,6 +178,10 @@
 			$pconfig['outgoing_interface'] = implode(",", $pconfig['outgoing_interface']);
 		}
 
+		if (isset($pconfig['system_domain_local_zone_type']) && !empty($pconfig['system_domain_local_zone_type'])) {
+			$display_system_domain_local_zone_type = $pconfig['system_domain_local_zone_type'];
```

Comment:
This variable is unnecessary, code makes sure $pconfig['system_domain_local_zone_type'] is never empty then you can use it at line 209 and just remove $display_system_domain_local_zone_type